src/Core/Style.js

```diff
@@ -160,6 +161,64 @@ function defineStyleProperty(style, category, name, value, defaultValue) {
     style[category][name] = value;
 }
 
+/**
+ * @class
+ * @classdesc StyleContext stores metadata of one FeatureGeometry that are needed for its style computation:
+ * type of feature and what is needed (fill, stroke or draw a point, etc.) as well as where to get its
+ * properties and its coordinates (for base_altitude).
+ *
+ * @property {Coordinates}      worldCoord @private Coordinates of the FeatureGeometry in world system.
+ * @property {Coordinates}      localCoordinates @private Are the coordinates systeme origin local or global.
+ * @property {boolean}          isProjected @private Are the coordinates already been projected.
```

Comment:
would be clearer if named `needToComputeWorldCoords` or `worldCoordsComputed`. Also, update the doc to `Have the world coordinates already been computed from the local coordinates`

Reply:
indeed, i did changed it (I was thinking about it too, but as it was the last modificaiton done by @Desplandis, i thiught i should not changed it)